SConstruct

```diff
@@ -13,7 +14,7 @@ import methods
 import gles_builders
 from platform_methods import run_in_subprocess
 
-# scan possible build platforms
+# Scan possible build platforms
```

Comment:
Could also take the opportunity to capitalize other comments, and add periods to the comments.

Reply:
No, cosmetic changes in unrelated parts of the file are bad practice.

Reply:
Well it *used* to be in the same code chunk as I had added code right below it, but that's no longer the case now.